packages/maskbook/src/plugins/ITO/UI/CreateForm.tsx

```diff
@@ -286,6 +321,26 @@ export function CreateForm(props: CreateFormProps) {
             <Box className={classes.date}>
                 {StartTime} {EndTime}
             </Box>
+            {account ? (
+                <Box className={classes.line}>
+                    <TextField
+                        className={classes.input}
+                        label={t('plugin_ito_qualification_label')}
+                        onChange={qualificationChange}
+                        value={qualification}
+                        InputLabelProps={{
+                            shrink: true,
+                        }}
+                        InputProps={{
+                            endAdornment: verify ? (
```

Comment:
This UI is not all state covered. Missing error and loading state.

Reply:
done